gulpfile.js

```diff
@@ -44,27 +43,19 @@ function updateWatchDist() {
     .pipe(gulp.dest(watchDist));
 }
 
-// transpiles a given SASS source set to CSS, storing results to libraryDist.
-function transpileSASS(src, debug) {
+// transpiles a given LESS source set to CSS, storing results to libraryDist.
+function transpileLESS(src, debug) {
   var opts = {
-    outputStyle: 'compressed',
-    includePaths: sass.modules.map(function (val) {
-      return val.sassPath;
-    })
-  };
-  if (debug) {
-    opts.outputStyle = 'expanded';
-    opts.sourceComments = true;
-    console.log('Compiling', src, 'in debug mode using SASS options:', opts);
+    // paths: [ path.join(__dirname, 'less', 'includes') ],
```

Comment:
you can remove this - if it is needed for reference, please include a comment as such.